### PR TITLE
Add the word custom

### PR DESCRIPTION
**Pull request by savoy9**
In line 95 a link to the PBI custom visuals doc has the plain text of "Power BI Visuals" in the context of saying they aren't all supported. this PR changes that text to Power BI custom visuals to make clear that 1st party visuals are 100% supported.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@savoy9 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/service-report-subscribe.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ A subscription ends if the Pro license expires, the owner deletes the dashboard
 * Report page subscriptions are tied to the name of the report page. If you subscribe to a report page and then rename it, you have to re-create your subscription.
 * Your organization may configure certain settings in Azure Active Directory that limit the ability to use email subscriptions in Power BI.  These limitations include, but aren't limited to, having multi-factor authentication or IP range restrictions when accessing resources.
 * Currently, email subscriptions for reports/dashboards using live connection datasets aren't supported when subscribing users other than yourself unless you are using a paginated report, which will allow you to send the subscription to others using your security context.
-* Email subscriptions don't support most [Power BI visuals](developer/visuals/power-bi-custom-visuals.md).  The one exception is those Power BI visuals that have been [certified](developer/visuals/power-bi-custom-visuals-certified.md).  
+* Email subscriptions don't support most [Power BI custom visuals](developer/visuals/power-bi-custom-visuals.md).  The one exception is those Power BI visuals that have been [certified](developer/visuals/power-bi-custom-visuals-certified.md).
```
Comment (maggiesMSFT) [inline powerbi-docs/service-report-subscribe.md]:
```suggestion
* Only default and [certified Power BI visuals](developer/visuals/power-bi-custom-visuals.md) are supported for email subscriptions.  
```

**Comment by PRMerger7** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
Hi, @savoy9 -- thanks for suggesting that change! It sparked a much needed conversation around terminology, which is still ongoing. In the meantime, I'm updating the article, not with your suggestion, but I hope capturing the spirit of your suggestion.
Thanks again!
#sign-off

**Pull request merged**